src/store/modules/chatMessages.js

```diff
@@ -0,0 +1,146 @@
+// Don't mind this import. I will use it in my next PR.
+// import ChatMessageService from '../../services/ChatMessage/ChatMessage';
+
+const state = [
+  {
+    chatId: 1,
+    senderName: "Virtual Concierge",
+    senderIcon: "logo.png",
+    messages: [
+      {
+        chatMessageId: 1,
+        receivedTime: Date.now(),
+        isUser: false,
+        text: "Oh... Well, this is akward.",
+      },
+      {
+        chatMessageId: 2,
+        receivedTime: Date.now(),
+        isUser: true,
+        text: "I meant I am working too hard; I am all fired up right now.",
+      },
+      {
+        chatMessageId: 3,
+        receivedTime: Date.now(),
+        isUser: false,
+        text: "I appreciate your concern. Apply water vigorously and call 911.",
+      },
+      {
+        chatMessageId: 4,
+        receivedTime: Date.now(),
+        isUser: true,
+        text: "I need help! My desk is on fire!",
+      },
+      {
+        chatMessageId: 5,
+        receivedTime: Date.now(),
+        isUser: false,
+        text: "Hi, how can I help you?",
+      },
+      {
+        chatMessageId: 6,
+        receivedTime: Date.now(),
+        isUser: false,
+        text: "Oh... Well, this is akward.",
+      },
+      {
+        chatMessageId: 7,
+        receivedTime: Date.now(),
+        isUser: true,
+        text: "I meant I am working too hard; I am all fired up right now.",
+      },
+      {
+        chatMessageId: 8,
+        receivedTime: Date.now(),
+        isUser: false,
+        text: "I appreciate your concern. Apply water vigorously and call 911.",
+      },
+      {
+        chatMessageId: 9,
+        receivedTime: Date.now(),
+        isUser: true,
+        text: "I need help! My desk is on fire!",
+      },
+      {
+        chatMessageId: 10,
+        receivedTime: Date.now(),
+        isUser: false,
+        text: "Hi, how can I help you?",
+      },
+    ],
+  },
+];
+
+// getters
+const getters = {
+  // Get virtual concierge items
+  getChatMessage(state) {
+    return state[0];
```

Comment:
We want to get messages in order of datetime

Reply:
This Get is to fetch the Conversation object. Not just the messages.
The Get will completely change once we have the inbox "selected item" functionality. 

Right now I hardcoded the first conversation [0] in the state. 

Also, getting the messages by date might not be needed as they are added to the array of messages in chronological order. Ordering can also be done in the front end. Remains to be seen..

Reply:
Ordering by index isn't a good idea. There is no guarantee it is in order, it could potentially be a alphanumeric guid. I'd recommend two getters. One that returns state[0] and one that returns state[0].messages in order

Reply:
I created the two methods in the getter requested.